### PR TITLE
doc: Add Pipefy to list of current companies using Argocd

### PR DESCRIPTION
**Pull request by yrosaguiar**
Checklist:

* [ ] Either (a) I've created an [enhancement proposal](https://github.com/argoproj/argo-cd/issues/new/choose) and discussed it with the community, (b) this is a bug fix, or (c) this does not need to be in the release notes.
* [ ] The title of the PR states what changed and the related issues number (used for the release note).
* [ ] I've updated both the CLI and UI to expose my feature, or I plan to submit a second PR with them.
* [X] Optional. My organization is added to the README.
* [X] I've signed the CLA and my build is green ([troubleshooting builds](https://argoproj.github.io/argo-cd/developer-guide/ci/)).

### CODE REVIEW

**Comment by alexec** [comment]
Thank you @yrosaguiar. I’m just trying to check your relationship with Pipefy, and couldn’t seem to find anything. Is there anything you could point me to please?

**Comment by yrosaguiar** [comment]
> Thank you @yrosaguiar. I’m just trying to check your relationship with Pipefy, and couldn’t seem to find anything. Is there anything you could point me to please?

Sure, the source branch is from Pipefy github, I'm SRE on Pipefy. You can verify my LinkedIn (https://www.linkedin.com/in/yros-pereira-aguiar-6a893b32/)  or I can send a email for you from Pipefy email domain

File: README.md [hunk 1]
```diff
@@ -44,6 +44,7 @@ Organizations below are **officially** using Argo CD. Please send a PR with your
 1. [Mirantis](https://mirantis.com/)
 1. [OpenSaaS Studio](https://opensaas.studio)
 1. [Optoro](https://www.optoro.com/)
+   [Pipefy](https://www.pipefy.com/)
```
Comment (alexmt) [inline README.md]:
```suggestion
1. [Pipefy](https://www.pipefy.com/)
```

**Comment by alexmt** [comment]
Thank you @yrosaguiar ! Please add `1. ` in front of company name (added change as a suggestion).

File: README.md (hunk 1 above)
Comment (yrosaguiar) [reply README.md]: Thanks, @alexmt  =)

**Review by alexmt — APPROVED** [review]

**Pull request merged**